gitgud/skills/util.py

```diff
@@ -225,30 +224,20 @@ def __setitem__(self, key, item):
         else:
             raise TypeError
 
-    def __contains__(self, key):
-        if isinstance(key, str):
-            if key.isnumeric():
-                return key in self.keys(str)
-            return key in self._name_dict.keys()
-        elif isinstance(key,  int):
-            return key in range(len(self._name_dict.keys()))
-        else:
-            return key in self._items
+    def __contains__(self, item):
+        return item in self._items
 
     def values(self):
         return self._items
     
-    def keys(self, keytype): # return the numbers, or return the names?
-        if keytype is str:
-            return {keytype(int(k) + 1) for k in self._name_dict.values()}
-        return {keytype(k) for k in self._name_dict.values()}
+    def keys(self): 
+        return set([str(int(k) + 1) for k in self._name_dict.values()]) | set(self._name_dict.keys())
```

Comment:
I think it would make sense to separate this out into three lines. The three lines would be setting a variable for the first set and a variable for the second set, then finally combining them in the return statement. The variable names should be descriptive

 Also instead of  using `self._name_dict.values`, why not use `range(len(self))` then do the i + 1 stuff. Also, use i instead of k since it's an iteration parameter, not a key.